### ISSUE TITLE
SSL Errors with delugePostProcess.py

### ISSUE THREAD

**Issue by anwnn** [mdhiggins/sickbeard_mp4_automator#1562]
After a recent pull of SMA, I'm now seeing SSL errors when the deluge post process script attempts to run. 

**Log files**
deluge.log:
```
16:24:52 [WARNING ][deluge_execute.core           :119 ] Command "/path/to/sickbeard_mp4_automator/delugePostProcess.py" failed with exit code 1
16:24:52 [WARNING ][deluge_execute.core           :121 ] stdout: 2022-04-04 16:24:52 - DelugePostProcess - INFO - Deluge post processing started.
2022-04-04 16:24:52 - resources.readsettings - INFO - /usr/local/bin/python3
2022-04-04 16:24:52 - resources.readsettings - INFO - Loading config file /path/to/sickbeard_mp4_automator/config/autoProcess.ini.
2022-04-04 16:24:52 - DelugePostProcess - ERROR - Unexpected exception.
Traceback (most recent call last):
  File "/path/to/sickbeard_mp4_automator/delugePostProcess.py", line 54, in <module>
    client.connect()
  File "/var/packages/py3k/target/usr/local/lib/python3.8/site-packages/deluge_client/client.py", line 84, in connect
    self._connect()
  File "/var/packages/py3k/target/usr/local/lib/python3.8/site-packages/deluge_client/client.py", line 98, in _connect
    self._socket.connect((self.host, self.port))
  File "/var/packages/py3k/target/usr/local/lib/python3.8/ssl.py", line 1342, in connect
    self._real_connect(addr, False)
  File "/var/packages/py3k/target/usr/local/lib/python3.8/ssl.py", line 1333, in _real_connect
    self.do_handshake()
  File "/var/packages/py3k/target/usr/local/lib/python3.8/ssl.py", line 1309, in do_handshake
    self._sslobj.do_handshake()
ssl.SSLError: [SSL: SSLV3_ALERT_HANDSHAKE_FAILURE] sslv3 alert handshake failure (_ssl.c:1124)
```

sma.log
```
2022-04-04 16:24:52 - DelugePostProcess - INFO - Deluge post processing started.
2022-04-04 16:24:52 - resources.readsettings - INFO - /usr/local/bin/python3
2022-04-04 16:24:52 - resources.readsettings - INFO - Loading config file /path/to/sickbeard_mp4_automator/config/autoProcess.ini.
2022-04-04 16:24:52 - DelugePostProcess - ERROR - Unexpected exception.
Traceback (most recent call last):
  File "/path/to/sickbeard_mp4_automator/delugePostProcess.py", line 54, in <module>
    client.connect()
  File "/var/packages/py3k/target/usr/local/lib/python3.8/site-packages/deluge_client/client.py", line 84, in connect
    self._connect()
  File "/var/packages/py3k/target/usr/local/lib/python3.8/site-packages/deluge_client/client.py", line 98, in _connect
    self._socket.connect((self.host, self.port))
  File "/var/packages/py3k/target/usr/local/lib/python3.8/ssl.py", line 1342, in connect
    self._real_connect(addr, False)
  File "/var/packages/py3k/target/usr/local/lib/python3.8/ssl.py", line 1333, in _real_connect
    self.do_handshake()
  File "/var/packages/py3k/target/usr/local/lib/python3.8/ssl.py", line 1309, in do_handshake
    self._sslobj.do_handshake()
ssl.SSLError: [SSL: SSLV3_ALERT_HANDSHAKE_FAILURE] sslv3 alert handshake failure (_ssl.c:1124)

```

**System Information**
 - OS: Synology DSM 6.2.4
 - Python 3.8.6
 - Deluge 2.0.3

**Expected behavior**
Nothing about my config has changed, it began occurring after a recent pull of the SMA git repo.

**Comment by mdhiggins** [comment]
b4ddcfcb52375ae78ee799140d5c88d6bc9e1bee

See if this helps, this was an attempt to fix an SSL error introduced in 3.10 that was supposed to be backwards compatible but I guess not, this will apply the fix conditionally

**Comment by anwnn** [comment]
Yup, that looks to have been the trick. 

Saw the earlier commit about SSL and 3.10, and _almost_ went down the path of upgrading Python. Thanks for the quick fix.
